packages/swarm-docs/src/pages/Radio.mdx

```diff
@@ -1,12 +1,53 @@
-import { Radio } from '@meetup/swarm-components'
-import Example from '../components/radioExample'
+import { Radio } from '@meetup/swarm-components';
+import Example from '../components/radioExample';
+import PropsTable from '../components/propstable';
 
-# Radio
+## Radio
+---
+A wrapper for an html `<input type="radio">`.
+
+<br/>
```

Comment:
Same question about `<br/>`s - is this trying to compensate for lack of margins/padding styles in the docs? Should we adjust styles instead?

Reply:
Yeah, same as above. No idea what the planning for styling the docs is. i'm assuming it's not super high priority, so <br/> tags are an easy solution for me for now

Reply:
OK, `<br>`s are not a blocker here, but we should create a ticket to fix styling for docs.